src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.html

```diff
@@ -0,0 +1,20 @@
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item">Cluster</li>
+    <li class="breadcrumb-item active">Pools</li>
+  </ol>
+</nav>
+<cd-table [data]="pools"
+          (fetchData)="getPoolList()"
+          [columns]="columns"
+          selectionType="single"
+          (updateSelection)="updateSelection($event)">
+  <tabset cdTableDetail *ngIf="selection.hasSingleSelection">
+    <tab i18n-heading
+         heading="Details">
+      <cd-table-key-value [data]="selection.first()" [autoReload]="undefined">
```

Comment:
Could you please use ``autoReload="false"``? That is much more telling than ``undefined``.

Reply:
done